### PR TITLE
[MIG] mgmtsystem_review: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**
Dependencies:
- [x] mgmtsystem_nonconformity: https://github.com/OCA/management-system/pull/372
- [x] mgmtsystem_survey: https://github.com/OCA/management-system/pull/367

### CODE REVIEW

**Comment by YvanDotet** [comment]
Hello
Is it ready to merge in the 14.0 branch ?
Thank you.

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+odoo14-addon-knowledge @git+https://github.com/dzungtran89/knowledge@14.0-mig-knowledge#subdirectory=setup/knowledge
```
Comment (simahawk) [inline test-requirements.txt]: to cleanup

**Review by simahawk — CHANGES_REQUESTED** [review]
LG

**Comment by gurneyalex** [comment]
as the dependencies are now merged, can you rebase to trigger a rebuild?

**Comment by dzungtran89** [comment]
Hi @gurneyalex , the PR has been updated

File: mgmtsystem_review/models/mgmtsystem_review.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 class MgmtsystemReview(models.Model):
     _name = "mgmtsystem.review"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
```
Comment (simahawk) [inline mgmtsystem_review/models/mgmtsystem_review.py:9]: for this change you should add the dependency on `mail` no?

Comment (hailangvn) [reply mgmtsystem_review/models/mgmtsystem_review.py:9]: Good day @simahawk, thanks for your comment. This module depends on `mgmtsystem_survey`, which depends on `survey`, and in turn, depends on `mail`. So we can keep this as is.

**Review by sebalix — APPROVED** [review]
LG

File: mgmtsystem_review/models/mgmtsystem_review.py (hunk 2 above)
Comment (simahawk) [reply mgmtsystem_review/models/mgmtsystem_review.py:9]: I always state such direct dependencies explicitly but that's just my pov ;)

**Review by simahawk — APPROVED** [review]

**Pull request merged**